tests/dummy/config/environment.js

```diff
@@ -3,14 +3,17 @@
 module.exports = function (environment) {
   var ENV = {
     modulePrefix: 'dummy',
-    podModulePrefix: 'dummy/pods',
```

Comment:
Sorry, on mobile so it's hard to check - is the demo not in pods structure with a `pods` directory?  If so this is necessary afaik